src/graphql/schema.ts

```diff
@@ -2,7 +2,13 @@ export const typeDefs = `#graphql
   type Query {
     hello: String
     getUser(id: Int!): User!
-    getManyUsers(limit: Int): [User!]! 
+    getManyUsers(offset: Int, limit: Int): UserPagination
+  }
+
+  type UserPagination {
+    totalCount: Int!
+    users: [User!]
+    isLastPage: Boolean
```

Comment:
```suggestion
    hasMoreUsers: Boolean
```

A diferença de nomenclatura é sútil, mas imagina se alguém manda um offset maior do que o total de usuários. Deveríamos avisar que não tem mais o que buscar depois disso, mas tecnicamente não se trata da última página, até porque ele transbordou.